packages/contracts-bedrock/scripts/upgrades/holocene/.env.example

```diff
@@ -0,0 +1,63 @@
+##############################################
+#               ↓  Required  ↓               #
+##############################################
+
+# Etherscan API key used to verify contract bytecode
+ETHERSCAN_API_KEY=
+
+# RPC URL for the L1 network that matches $NETWORK
+ETH_RPC_URL=
+
+# Private key used to deploy the new contracts for this upgrade
+PRIVATE_KEY=
+
+# Path to the deploy config JSON file
+DEPLOY_CONFIG_PATH=
+
+# Path to the folder where output artifacts will be stored
+OUTPUT_FOLDER_PATH=
+
+# Address of the deployed `SystemConfig` implementation for Holocene.
```

Comment:
Can we move all of these to a section near the bottom of the `.env.example` with a big header that says "don't set these values unless you know what you're doing"?

Reply:
sure